src/sqlfluff/dialects/dialect_postgres.py

```diff
@@ -911,11 +911,9 @@ class CreateTableStatementSegment(BaseSegment):
                         AnyNumberOf(
                             Sequence(
                                 Ref("ParameterNameSegment"),
-                                Sequence(
-                                    Ref("EqualsSegment"),
-                                    Ref("LiteralGrammar"),
-                                    optional=True,
-                                ),
+                                Ref("EqualsSegment"),
+                                Ref("LiteralGrammar"),
+                                optional=True,
```

Comment:
I've just reread the docs, and realised that asking you to change this here and in createtableas was bad advice 🤦‍♂️ . Can you change it back? And then I think we're good to go.

Reply:
Done!